### DISCUSSION TITLE
Aqara Sensore Illuminazione - parametri mancanti

### DISCUSSION THREAD

**Discussion by baldus88** [indomus/forum#2689] [ZigBee, Z-Wave, Thread, Matter]
Ciao ragazzi.

Vi porto all'attenzione un'anomalia del mio sensore di illuminazione Aqara. In poche parole da un giorno all'altro (senza che io aggiornassi Home assistant o Zigbee2mqtt) ha smesso di aggiornarsi il valore "illuminance" sul sensore di Home Assistant, me ne sono accorto che non mi andava più l'automazione.
Anche se dal dispositivo si continua a vedere tutto ok.

![image](https://github.com/indomus/forum/assets/57867040/37d2b506-9d0b-4dcc-9984-486181b3c9a6)

Ho provato a toglierlo del tutto dal portale zigbee2mqtt e riaggiungerlo e mi vede solo il valore lux:

![image](https://github.com/indomus/forum/assets/57867040/c926d11d-cdc3-4c55-8561-2bebe0ced599)

Questi sono i log di errore che genera quando prova a fare delle azioni:
`Error 2023-11-03 01:19:46Publish 'get' 'illuminance_lux' to 'bagno_illum' failed: 'Error: Read 0x04cf8cdf3c7f2163/1 msIlluminanceMeasurement(["measuredValue"], {"sendWhen":"immediate","timeout":10000,"disableResponse":false,"disableRecovery":false,"disableDefaultResponse":true,"direction":0,"srcEndpoint":null,"reservedBits":0,"manufacturerCode":null,"transactionSequenceNumber":null,"writeUndiv":false}) failed (Timeout - 22517 - 1 - 48 - 1024 - 1 after `10000ms)'``

Leggo timeout ma di cosa secondo voi?

### Zigbee2MQTT version

1.33.2 commit: 9996c93

### Home Assisant version
2023.9.1

### Adapter firmware version

20200211

### Adapter

zStack30x

### Debug log in quella fascia dell'errore in modalità debug

``Info 2023-11-03 01:15:11MQTT publish: topic 'zigbee2mqtt/sensore_bagno', payload '{"battery":97,"device":{"applicationVersion":3,"dateCode":"20161129","friendlyName":"sensore_bagno","hardwareVersion":30,"ieeeAddr":"0x00158d0003f0d630","manufacturerID":4151,"manufacturerName":"LUMI","model":"WSDCGQ11LM","networkAddress":8577,"powerSource":"Battery","softwareBuildID":"3000-0001","stackVersion":2,"type":"EndDevice","zclVersion":1},"humidity":70.46,"linkquality":31,"power_outage_count":59203,"pressure":967.6,"temperature":21.15,"voltage":2995}'
Info 2023-11-03 01:15:56MQTT publish: topic 'zigbee2mqtt/sensore_cucina', payload '{"battery":23,"device":{"applicationVersion":3,"dateCode":"20161129","friendlyName":"sensore_cucina","hardwareVersion":30,"ieeeAddr":"0x00158d000411b47f","manufacturerID":4151,"manufacturerName":"LUMI","model":"WSDCGQ11LM","networkAddress":31792,"powerSource":"Battery","softwareBuildID":"3000-0001","stackVersion":2,"type":"EndDevice","zclVersion":1},"humidity":69.99,"linkquality":68,"power_outage_count":28252,"pressure":967,"temperature":21.37,"voltage":2885}'
Info 2023-11-03 01:15:56MQTT publish: topic 'zigbee2mqtt/sensore_cucina', payload '{"battery":23,"device":{"applicationVersion":3,"dateCode":"20161129","friendlyName":"sensore_cucina","hardwareVersion":30,"ieeeAddr":"0x00158d000411b47f","manufacturerID":4151,"manufacturerName":"LUMI","model":"WSDCGQ11LM","networkAddress":31792,"powerSource":"Battery","softwareBuildID":"3000-0001","stackVersion":2,"type":"EndDevice","zclVersion":1},"humidity":70.06,"linkquality":63,"power_outage_count":28252,"pressure":967,"temperature":21.37,"voltage":2885}'
Info 2023-11-03 01:15:56MQTT publish: topic 'zigbee2mqtt/sensore_cucina', payload '{"battery":23,"device":{"applicationVersion":3,"dateCode":"20161129","friendlyName":"sensore_cucina","hardwareVersion":30,"ieeeAddr":"0x00158d000411b47f","manufacturerID":4151,"manufacturerName":"LUMI","model":"WSDCGQ11LM","networkAddress":31792,"powerSource":"Battery","softwareBuildID":"3000-0001","stackVersion":2,"type":"EndDevice","zclVersion":1},"humidity":70.06,"linkquality":63,"power_outage_count":28252,"pressure":967.3,"temperature":21.37,"voltage":2885}'
Info 2023-11-03 01:16:17Configuring 'bagno_illum'
Error 2023-11-03 01:16:27Failed to configure 'bagno_illum', attempt 1 (Error: Bind 0x04cf8cdf3c7f2163/1 msIlluminanceMeasurement from '0x00124b0009d6caa1/1' failed (AREQ - ZDO - bindRsp after 10000ms) at Timeout._onTimeout (/app/node_modules/zigbee-herdsman/src/utils/waitress.ts:64:35) at listOnTimeout (node:internal/timers:569:17) at processTimers (node:internal/timers:512:7))
Info 2023-11-03 01:16:27MQTT publish: topic 'zigbee2mqtt/bridge/response/device/configure', payload '{"data":{"id":"bagno_illum"},"error":"Failed to configure (Bind 0x04cf8cdf3c7f2163/1 msIlluminanceMeasurement from '0x00124b0009d6caa1/1' failed (AREQ - ZDO - bindRsp after 10000ms))","status":"error","transaction":"rfdsn-1"}'
Info 2023-11-03 01:18:11Successfully changed options
Info 2023-11-03 01:18:11MQTT publish: topic 'zigbee2mqtt/bridge/response/options', payload '{"data":{"restart_required":false},"status":"ok","transaction":"rfdsn-2"}'
Debug 2023-11-03 01:18:12Received Zigbee message from 'sensore_sala', type 'attributeReport', cluster 'msTemperatureMeasurement', data '{"measuredValue":2151}' from endpoint 1 with groupID 0
Info 2023-11-03 01:18:12MQTT publish: topic 'zigbee2mqtt/sensore_sala', payload '{"battery":100,"device":{"applicationVersion":3,"dateCode":"20161129","friendlyName":"sensore_sala","hardwareVersion":30,"ieeeAddr":"0x00158d0003fb07fd","manufacturerID":4151,"manufacturerName":"LUMI","model":"WSDCGQ11LM","networkAddress":62317,"powerSource":"Battery","softwareBuildID":"3000-0001","stackVersion":2,"type":"EndDevice","zclVersion":1},"humidity":68.27,"linkquality":81,"power_outage_count":17644,"pressure":967.1,"temperature":21.51,"voltage":3025}'
Debug 2023-11-03 01:18:12Received Zigbee message from 'sensore_sala', type 'attributeReport', cluster 'msRelativeHumidity', data '{"measuredValue":6826}' from endpoint 1 with groupID 0
Debug 2023-11-03 01:18:12Received Zigbee message from 'sensore_sala', type 'attributeReport', cluster 'msPressureMeasurement', data '{"measuredValue":967,"scale":-1,"scaledValue":9673}' from endpoint 1 with groupID 0
Info 2023-11-03 01:18:12MQTT publish: topic 'zigbee2mqtt/sensore_sala', payload '{"battery":100,"device":{"applicationVersion":3,"dateCode":"20161129","friendlyName":"sensore_sala","hardwareVersion":30,"ieeeAddr":"0x00158d0003fb07fd","manufacturerID":4151,"manufacturerName":"LUMI","model":"WSDCGQ11LM","networkAddress":62317,"powerSource":"Battery","softwareBuildID":"3000-0001","stackVersion":2,"type":"EndDevice","zclVersion":1},"humidity":68.26,"linkquality":81,"power_outage_count":17644,"pressure":967.1,"temperature":21.51,"voltage":3025}'
Info 2023-11-03 01:18:12MQTT publish: topic 'zigbee2mqtt/sensore_sala', payload '{"battery":100,"device":{"applicationVersion":3,"dateCode":"20161129","friendlyName":"sensore_sala","hardwareVersion":30,"ieeeAddr":"0x00158d0003fb07fd","manufacturerID":4151,"manufacturerName":"LUMI","model":"WSDCGQ11LM","networkAddress":62317,"powerSource":"Battery","softwareBuildID":"3000-0001","stackVersion":2,"type":"EndDevice","zclVersion":1},"humidity":68.26,"linkquality":81,"power_outage_count":17644,"pressure":967.3,"temperature":21.51,"voltage":3025}'
Debug 2023-11-03 01:18:22Received Zigbee message from 'sensore_movimento_bagno', type 'commandDataResponse', cluster 'manuSpecificTuya', data '{"dpValues":[{"data":{"data":[1],"type":"Buffer"},"datatype":4,"dp":3}],"seq":27393}' from endpoint 1 with groupID 0
Info 2023-11-03 01:18:22MQTT publish: topic 'zigbee2mqtt/sensore_movimento_bagno', payload '{"device":{"applicationVersion":73,"dateCode":"20180727","friendlyName":"sensore_movimento_bagno","hardwareVersion":1,"ieeeAddr":"0xec1bbdfffe303275","manufacturerID":0,"manufacturerName":"_TYST11_i5j6ifxj","model":"BW-IS3","networkAddress":3526,"powerSource":"Battery","stackVersion":0,"type":"EndDevice","zclVersion":3},"linkquality":21,"occupancy":true}'
Debug 2023-11-03 01:18:30Received Zigbee message from 'sensore_movimento_bagno', type 'commandDataResponse', cluster 'manuSpecificTuya', data '{"dpValues":[{"data":{"data":[1],"type":"Buffer"},"datatype":4,"dp":3}],"seq":27393}' from endpoint 1 with groupID 0
Info 2023-11-03 01:18:30MQTT publish: topic 'zigbee2mqtt/sensore_movimento_bagno', payload '{"device":{"applicationVersion":73,"dateCode":"20180727","friendlyName":"sensore_movimento_bagno","hardwareVersion":1,"ieeeAddr":"0xec1bbdfffe303275","manufacturerID":0,"manufacturerName":"_TYST11_i5j6ifxj","model":"BW-IS3","networkAddress":3526,"powerSource":"Battery","stackVersion":0,"type":"EndDevice","zclVersion":3},"linkquality":0,"occupancy":true}'
Debug 2023-11-03 01:18:59Received MQTT message on 'zigbee2mqtt/bagno_illum/get' with data '{"illuminance":""}'
Debug 2023-11-03 01:18:59Publishing get 'get' 'illuminance' to 'bagno_illum'
Debug 2023-11-03 01:19:01Received MQTT message on 'zigbee2mqtt/bagno_illum/get' with data '{"illuminance_lux":""}'
Debug 2023-11-03 01:19:01Publishing get 'get' 'illuminance_lux' to 'bagno_illum'
Debug 2023-11-03 01:19:03Received MQTT message on 'zigbee2mqtt/luce_bagno/set' with data '{"state":"ON"}'
Debug 2023-11-03 01:19:03Publishing 'set' 'state' to 'luce_bagno'
Info 2023-11-03 01:19:04MQTT publish: topic 'zigbee2mqtt/luce_bagno', payload '{"brightness":254,"device":{"applicationVersion":16,"dateCode":"20140331CNEF****","friendlyName":"luce_bagno","hardwareVersion":1,"ieeeAddr":"0x7cb03eaa0a0436ac","manufacturerID":48042,"manufacturerName":"OSRAM","model":"AC03641","networkAddress":13011,"powerSource":"Mains (single phase)","softwareBuildID":"V1.05.10","stackVersion":2,"type":"Router","zclVersion":1},"linkquality":5,"state":"ON","update":{"installed_version":16909584,"latest_version":16909584,"state":"idle"},"update_available":null}'
Debug 2023-11-03 01:19:10Received Zigbee message from 'bagno_illum', type 'attributeReport', cluster 'msIlluminanceMeasurement', data '{"measuredValue":18326}' from endpoint 1 with groupID 0
Info 2023-11-03 01:19:10MQTT publish: topic 'zigbee2mqtt/bagno_illum', payload '{"battery":100,"device":{"applicationVersion":26,"dateCode":"20200305","friendlyName":"bagno_illum","hardwareVersion":1,"ieeeAddr":"0x04cf8cdf3c7f2163","manufacturerID":4718,"manufacturerName":"XIAOMI","model":"GZCGQ01LM","networkAddress":22517,"powerSource":"Battery","softwareBuildID":"2019\u0000www.","stackVersion":2,"type":"EndDevice","zclVersion":3},"illuminance":18326,"illuminance_lux":68,"linkquality":7,"voltage":3100}'
Debug 2023-11-03 01:19:10Received MQTT message on 'zigbee2mqtt/luce_bagno/set' with data '{"state":"OFF"}'
Debug 2023-11-03 01:19:10Publishing 'set' 'state' to 'luce_bagno'
Info 2023-11-03 01:19:10MQTT publish: topic 'zigbee2mqtt/luce_bagno', payload '{"brightness":254,"device":{"applicationVersion":16,"dateCode":"20140331CNEF****","friendlyName":"luce_bagno","hardwareVersion":1,"ieeeAddr":"0x7cb03eaa0a0436ac","manufacturerID":48042,"manufacturerName":"OSRAM","model":"AC03641","networkAddress":13011,"powerSource":"Mains (single phase)","softwareBuildID":"V1.05.10","stackVersion":2,"type":"Router","zclVersion":1},"linkquality":7,"state":"OFF","update":{"installed_version":16909584,"latest_version":16909584,"state":"idle"},"update_available":null}'
Info 2023-11-03 01:19:15MQTT publish: topic 'zigbee2mqtt/sensore_movimento_bagno', payload '{"device":{"applicationVersion":73,"dateCode":"20180727","friendlyName":"sensore_movimento_bagno","hardwareVersion":1,"ieeeAddr":"0xec1bbdfffe303275","manufacturerID":0,"manufacturerName":"_TYST11_i5j6ifxj","model":"BW-IS3","networkAddress":3526,"powerSource":"Battery","stackVersion":0,"type":"EndDevice","zclVersion":3},"linkquality":0,"occupancy":false}'
Error 2023-11-03 01:19:22Publish 'get' 'illuminance' to 'bagno_illum' failed: 'Error: Read 0x04cf8cdf3c7f2163/1 msIlluminanceMeasurement(["measuredValue"], {"sendWhen":"immediate","timeout":10000,"disableResponse":false,"disableRecovery":false,"disableDefaultResponse":true,"direction":0,"srcEndpoint":null,"reservedBits":0,"manufacturerCode":null,"transactionSequenceNumber":null,"writeUndiv":false}) failed (Timeout - 22517 - 1 - 47 - 1024 - 1 after 10000ms)'
Debug 2023-11-03 01:19:22Error: Read 0x04cf8cdf3c7f2163/1 msIlluminanceMeasurement(["measuredValue"], {"sendWhen":"immediate","timeout":10000,"disableResponse":false,"disableRecovery":false,"disableDefaultResponse":true,"direction":0,"srcEndpoint":null,"reservedBits":0,"manufacturerCode":null,"transactionSequenceNumber":null,"writeUndiv":false}) failed (Timeout - 22517 - 1 - 47 - 1024 - 1 after 10000ms) at Timeout._onTimeout (/app/node_modules/zigbee-herdsman/src/utils/waitress.ts:64:35) at listOnTimeout (node:internal/timers:569:17) at processTimers (node:internal/timers:512:7)
Debug 2023-11-03 01:19:25Received Zigbee message from 'bagno_illum', type 'attributeReport', cluster 'msIlluminanceMeasurement', data '{"measuredValue":0}' from endpoint 1 with groupID 0
Info 2023-11-03 01:19:25MQTT publish: topic 'zigbee2mqtt/bagno_illum', payload '{"battery":100,"device":{"applicationVersion":26,"dateCode":"20200305","friendlyName":"bagno_illum","hardwareVersion":1,"ieeeAddr":"0x04cf8cdf3c7f2163","manufacturerID":4718,"manufacturerName":"XIAOMI","model":"GZCGQ01LM","networkAddress":22517,"powerSource":"Battery","softwareBuildID":"2019\u0000www.","stackVersion":2,"type":"EndDevice","zclVersion":3},"illuminance":0,"illuminance_lux":0,"linkquality":10,"voltage":3100}'
Debug 2023-11-03 01:19:31Received MQTT message on 'zigbee2mqtt/bridge/request/options' with data '{"options":{"advanced":{"log_level":"info"}},"transaction":"rfdsn-3"}'
Info 2023-11-03 01:19:31Successfully changed options
Info 2023-11-03 01:19:31MQTT publish: topic 'zigbee2mqtt/bridge/response/options', payload '{"data":{"restart_required":false},"status":"ok","transaction":"rfdsn-3"}'
Error 2023-11-03 01:19:46Publish 'get' 'illuminance_lux' to 'bagno_illum' failed: 'Error: Read 0x04cf8cdf3c7f2163/1 msIlluminanceMeasurement(["measuredValue"], {"sendWhen":"immediate","timeout":10000,"disableResponse":false,"disableRecovery":false,"disableDefaultResponse":true,"direction":0,"srcEndpoint":null,"reservedBits":0,"manufacturerCode":null,"transactionSequenceNumber":null,"writeUndiv":false}) failed (Timeout - 22517 - 1 - 48 - 1024 - 1 after 10000ms)``

**Comment by KC-inDomus** [comment]
possibile che sia la batteria, oppure s'è guastato.
